README.md

```diff
@@ -1,24 +1,158 @@
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone git@github.com:ger619/ruby-capstone.git
+
+```
+
+$ gem install bundler
+
+```
+$ bundle install
+
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+Please run this command to start the application.
+```
+$ ruby main.rb
+```
+Also, a folder named `data` will be generated automatically once you exit the app.
+This folder will contain all the json files that are being read and write for data.
+
+### Run tests
+
+To install rspec, in the terminal kindly run this command
+
+```
+$ gem install rspec
+```
+
+To run tests, please run this command
+
+```
+$ rspec ./spec/#{filename}_spec.rb
+```
+
+## Authors
+
+👤 **Mavericks Balitaan** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-mavericks--db-white?logo=LinkedIn&logoColor=0A66C2&style=plastic)](https://www.linkedin.com/in/mavericks-db/)
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-mavericks--db-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/mavericks-db)
+
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-mavericks__db-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/mavericks_db)
+
+👤 **Nii Amanor Djoleto** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-nii--amanor--djoleto-white?logo=LinkedIn&logoColor=0A66C2&style=plastic)](https://linkedin.com/in/nii-amanor-djoleto)
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-ramrod433-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/ramrod433)
+
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-__Djoleto-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/_djoleto_)
+
+👤 **Abol Ger** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-David%20Ger-white?logo=LinkedIn&logoColor=0A66C2&style=plastic)](https://linkedin.com/in/david-ger-426b4576)
+
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-ger619-white?logo=GitHub&logoColor=181717&style=plastic)](https://github.com/ger619)
+
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-ger__abol-white?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/ger_abol)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ger619/ruby-capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://www.microverse.org/)
+- Coding Partners
+- Code Reviewers
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] OPTIONAL: Please, consider updating this link, with a `./MIT.md` file included at the root of your project, containing a proper MIT license.  

## Explanation

This license link does not work, because you are pointing to a `./MIT.md` file in the root your project that does not exist.   


## Tips:

[You can go here](https://github.com/microverseinc/readme-template):

![image](https://user-images.githubusercontent.com/57421823/192573418-dd344b81-5f0c-49ae-b2c4-1aab641151c1.png)

Copy the MIT.md to the root of your project.